src/main/java/ch/uzh/ifi/hase/soprafs21/service/UserService.java

```diff
@@ -125,4 +122,8 @@ public boolean isRequesterAndAuthenticatedUserTheSame(Long senderId) {
         return authenticatedUser.equals(sender);
     }
 
+    @Transactional
+    public void updateUserLocation(Long userId, Point newLocation) {
+        this.userRepository.updateUserLocation(userId, newLocation);
+    }
```

Comment:
...or rather, put the authorization I mentioned in the controller here?

Reply:
It will be added here when the security is ready.